mono/metadata/class.c

```diff
@@ -6615,10 +6615,13 @@ mono_class_from_name (MonoImage *image, const char* name_space, const char *name
 		if (res) {
 			if (!class)
 				class = search_modules (image, name_space, name);
```

Comment:
I think we are missing the case when `class` is `NULL` and `search_modules` returns `NULL`. Previously this code would return `NULL`. Now it will continue.